examples/rosetta/julia.wyv

```diff
@@ -0,0 +1,52 @@
+require stdout
+
+def iterate(a_z : Float, b_z : Float, a : Float, b : Float, c : Int) : String
+	(c > 200).ifTrue(
```

Comment:
Check out the "if" syntax used in fibonacci.wyv  I think it would make this code look a bit nicer.